src/drivers/driver.ts

```diff
@@ -82,16 +89,29 @@ export abstract class CMakeDriver implements vscode.Disposable {
     return Promise.resolve(true);
   }
 
+  /**
+   * Check if using cached configuration is supported.
+   */
+  protected abstract isCacheConfigSupported(): boolean;
```

Comment:
nit: you can make this a 'get' property so that you don't have to access it like a function (i.e. no parenthesis)